lib/X25519KeyAgreementKey2020.js

```diff
@@ -159,7 +186,7 @@ export class X25519KeyAgreementKey2020 extends LDKeyPair {
       throw new Error('Source public key is required to convert.');
     }
 
-    const edPubkeyBytes = _multibaseDecode({text: publicKeyMultibase});
+    const edPubkeyBytes = _multibaseDecode(publicKeyMultibase);
```

Comment:
This doesn't check the key header.

Reply:
Fixed in 5f79c9c90b7aa35f1d4ea5c889506b0b26c9961e.